docs/api/v3.rst

```diff
@@ -0,0 +1,487 @@
+
+   https://readthedocs.org/api/v2/project/?slug=pip
+
+
+Project import
+++++++++++++++
+
+.. http:post::  /api/v3/project/import/
+
+    Import a new project.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "name": "Pip",
+            "repo_url": "https://github.com/pypa/pip",
+            "repo_type": "git"
+            "description": "Pip Installs Packages.",
+            "documentation_type": "sphinx_htmldir",
+            "language": "en",
+            "programing_language": "py",
+            "tags": [
+                "pip",
+                "python",
+                "packaging"
+            ],
+            "homepage": "https://pip.readthedocs.io/"
+        }
+
+    ``name``, ``repo_url`` and ``repo_type`` are required.
+
+    **Example response**:
+
+    *See Project details*
+
+    :statuscode 201: Created sucessfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+Project edit
+++++++++++++
+
+.. http:patch::  /api/v3/project/(string:slug)/
+
+    Edit a project.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "description": "Pip helps you to install packages.",
+            "default_version": "stable",
+            "homepage": "https://pypi.org/project/pip/"
+        }
+
+    :statuscode 204: Edited successfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+Add User to Project
++++++++++++++++++++
+
+
+.. http:post::  /api/v3/project/(string:slug)/users/
+
+    Add a User as mantainer to the project.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "username": "humitos"
+        }
+
+    :statuscode 201: Added successfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+Add Domain to Project
++++++++++++++++++++++
+
+
+.. http:post::  /api/v3/project/(string:slug)/domains/
+
+    Add a Domain to the project.
+
+    **Example request**:
+
+    .. sourcecode:: js
+
+        {
+            "domain": "docs.pip.org"
+            "canonical": true,
+            "use_https": true
+        }
+
+    :statuscode 201: Added successfully
+    :statuscode 400: Some field is invalid
+    :statuscode 401: Not valid permissions
+
+
+Versions
+~~~~~~~~
+
+Versions are different versions of the same project documentation.
+
+The versions for a given project can be viewed in a project's version screen.
+For example, here is the `Pip project's version screen`_.
+
+.. _Pip project's version screen: https://readthedocs.org/projects/pip/versions/
+
+
+Version detail
+++++++++++++++
+
+.. http:get::  /api/v2/project/(string:project_slug)/version/(string:version_slug)/
+
+    Retrieve details of a single version.
+
+    .. sourcecode:: js
+
+        {
+            "slug": "stable",
+            "verbose_name": "stable",
+            "identifier": "3a6b3995c141c0888af6591a59240ba5db7d9914",
+            "built": true,
```

Comment:
I'd like to have a calculated field here: `build_date` or `last_success_build_date` or something with that meaning but a better name ;)